src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -321,9 +458,9 @@ export class StudyViewPageStore {
     }
 
     readonly clinicalAttributes = remoteData({
-        await: () => [this.studies],
+        await: () => [this.filteredVirtualStudies],
         invoke: () => defaultClient.fetchClinicalAttributesUsingPOST({
-            studyIds: this.studies.result.map(study => study.studyId)
+            studyIds: this.queriedPhysicalStudies
```

Comment:
Why we wait for filteredVirtualStudies when the API call is only on filteredVirtualStudies?